src/test/java/ch/uzh/ifi/fs22/sel/group15/cozycave/server/service/ApplicationServiceTest.java

```diff
@@ -0,0 +1,163 @@
+package ch.uzh.ifi.fs22.sel.group15.cozycave.server.service;
+
+import ch.uzh.ifi.fs22.sel.group15.cozycave.server.entity.Location;
+import ch.uzh.ifi.fs22.sel.group15.cozycave.server.entity.users.AuthenticationData;
+import ch.uzh.ifi.fs22.sel.group15.cozycave.server.entity.users.User;
+import ch.uzh.ifi.fs22.sel.group15.cozycave.server.entity.users.UserDetails;
+import ch.uzh.ifi.fs22.sel.group15.cozycave.server.repository.UserRepository;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.springframework.web.server.ResponseStatusException;
+
+class ApplicationServiceTest {
```

Comment:
next time: just don't add the test in the implementation phase yet instead of comment everything out